src/doc/trpl/arrays-vectors-and-slices.md

```diff
@@ -49,7 +49,7 @@ languages.
 
 A *vector* is a dynamic or "growable" array, implemented as the standard
 library type [`Vec<T>`](../std/vec/) (we'll talk about what the `<T>` means
-later). Vectors always allocate their data on the heap. Vectors are to slices 
+later). Vectors always allocate their data on the heap. Vectors are to arrays
```

Comment:
We should probably move addressing this part into another patch then—by your clarifying explanation this has several fundamental problems as it stands:

- Ownership hasn't been introduced at all by this stage, it comes later in the intermediate section of the book, so that analogy is completely lost.
- As I noted before, there is is no other use of the term "slice" in the guides up to this point except for "string slice".

   The unqualified term "slice" is introduced later in this chapter, after the analogy line we're discussing. The only logical assumption the reader can make about the analogy at that point is that it refers to string slices, and that makes no sense.

Possibly the part of this chapter that defines slices could be moved ahead of the part on vectors and that would help, although the line that says

> You can also take a slice of a vector, `String`, or `&str`...

should be reworked to not mention vectors before they're defined, but state that slices of vectors are possible later in the vectors part.

This doesn't help with the ownership analogy being lost on the reader, though.